jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -143,9 +145,19 @@ def _get_region_from_subset(self, subset):
             if subset_grp.label == subset:
                 for sbst in subset_grp.subsets:
                     if sbst.data.label == self.dataset_selected:
-                        # TODO: https://github.com/glue-viz/glue-astronomy/issues/52
-                        return sbst.data.get_selection_definition(
-                                subset_id=subset, format='astropy-regions')
+                        reg = sbst.data.get_selection_definition(
+                            subset_id=subset, format='astropy-regions')
+                        # Works around https://github.com/glue-viz/glue-astronomy/issues/52
+                        # Assume it is always pixel region, not sky region. Even with multiple
+                        # viewers, they all seem to share the same reference image even when it is
+                        # not loaded in all the viewers, so use default viewer.
+                        viewer = self.session.jdaviz_app._jdaviz_helper.default_viewer

```

Comment:
```suggestion
                        viewer = self.app._jdaviz_helper.default_viewer

```

(maybe we should provide direct access to the helper or default viewer from the `TemplateMixin`... but we can do that later)